Package/Sublime Text Syntax Definition/syntax_test_onig-regexp

```diff
@@ -471,6 +488,6 @@ where escape characters are ignored.\).
 (?-x)
 
 check if all scopes are cleared:
-# <- meta.literal - meta meta
+# <- (meta.literal) | - meta
```

Comment:
I think I didn't mean to change that test, not sure where it came from - I've undone the change now